kibana-notebooks/public/components/notebook.tsx

```diff
@@ -696,7 +734,7 @@ export class Notebook extends Component<NotebookProps, NotebookState> {
                           iconType='arrowDown'
                           iconSide='right'
                           onClick={() => this.setState({ isParaActionsPopoverOpen: true })}
-                        >Paragraph actions</EuiButton>
+                        >Actions</EuiButton>
```

Comment:
Could you not change these, the changes were made during reviews, so it's not consistent with design for sql/ppl. but they have nothing to do with sql/ppl functionality

Reply:
Yeah- will remove these and other UX changes left over in this PR